test/functionalTest/cases/117_convop_using_standard_ops/postAllEntitiesWithTypeAndId.test

```diff
@@ -672,4 +672,4 @@ Date: REGEX(.*)
 
 --TEARDOWN--
 brokerStop CB
-# dbDrop CB
+dbDrop CB
```

Comment:
Even better than '--noTeardown':

    function dbDrop()
    {
      db=$1

      if [ "$CB_DB_DROP" != "No" ]
      then
        if [ "$db" != "" ]
        then
          dbInit $db
        fi
      fi
    }

[ Already modified for my next PR (instead of creating an issue about it - it was faster! :-)) ]

To use it:

    CB_DB_DROP=No testHarness xxx.test